ServiceProviders/providers.js

```diff
@@ -9,14 +9,20 @@ let matchers = {
 
 	'youtube' : new youtubeProvider(),
 	'pdf': new pdfProvider(),
-	'docs': new docsProvider()
+	'docs': new docsProvider(),
+	
 }
 
 module.exports = {
 
 	run(win) {
 
-		let text = clipboard.readText('selection')
+		if(arguments[1]) {
+			clipboard.writeText(arguments[1])
```

Comment:
I think we should writing anything to the clipboard. This will replace anything already present on Clipboard.